tests/system/Log/LoggerTest.php

```diff
@@ -63,7 +64,7 @@ public function testLogActuallyLogs(): void
         $config = new LoggerConfig();
         $logger = new Logger($config);
 
-        $expected = 'DEBUG - ' . date('Y-m-d') . ' --> Test message';
+        $expected = 'DEBUG - ' . Time::now()->toDateString() . ' --> Test message';
```

Comment:
`toDateString()` returns localized version of date.
https://codeigniter4.github.io/CodeIgniter4/libraries/time.html
You should use `format('Y-m-d')`.